src/style.css

```
@@ -1 +1,15 @@
-/* styles go here */
+.input {
+  margin-top: 10px;
+}
+
+.fieldset {
+  margin: 20px 2px;
```

```suggestion
  margin-top: 20px;
```
margin bottom doesn't work anyway due to the [margin-collaps](https://www.w3schools.com/css/css_margin_collapse.asp)